src/main/scala/workshop4/assignments/FileSystemEventObserver.scala

```diff
@@ -0,0 +1,71 @@
+package workshop4.assignments
+
+import java.io.File
+import java.nio.file.Path
+import java.util
+
+import rx.Observer
```

Comment:
Went through all kind of incompatible parameter issues with the onNext as in the java test class. Didn't retry after I got it working with extending the Observer class.